vaadin-grid-flow/src/test/java/com/vaadin/flow/component/grid/dataview/GridListDataViewTest.java

```diff
@@ -125,6 +124,39 @@ public void dataView_withFilter_methodsUseFilteredData() {
                 dataView.getItemOnRow(0));
     }
 
+    @Test
+    public void dataViewWithItems_contains_returnsCorrectItems() {
+        Grid<String> grid = new Grid<>();
+        GridListDataView<String> dataView = grid.setItems("first","middle","last");
+
+        Assert.assertTrue("Returned false for item that should exist",
+                dataView.contains("middle"));
+        Assert.assertFalse("Returned false for item that should exist",
```

Comment:
This test case is already covered by `contains_itemNotPresentedInDataSet_itemNotFound `. My suggestion is to delete this assertion.